src/content/docs/start-here/inventory-api.md

```diff
@@ -31,13 +31,13 @@ Required data is marked with an asterisk (*).
 This object represents the attributes that are common for all resources, no matter the type.  Most of the metadata is generated by the Asset Inventory service, i.e.
 - ___id___ - this is the GUID for the resource instance
 - ___resource_type___ - this is derived from the top level object of the request body.
-- ___first_reported___
-- ___last_reported___
-- ___first_reported_by___
-- ___last_reported_by___
+- ___created_at___
+- ___updated_at___
+- ___deleted_at___
    
-The only metadata that a Reporter can provide is: 
-- ___workspace*___
+The only metadata that a Reporter provides is: 
+- ___org_id*___
```

Comment:
Are also including the `org_id` in the relationship metadata?

Reply:
good catch.  FYI...I moved some things around to 1) mention what the reporter provides first followed by what the service adds (was inconsistent across objects) and 2) added org_id to relationship type along with some rewording.